third_party/nvfuser/csrc/scheduler/reduction_heuristic.h

```diff
@@ -12,6 +12,10 @@ namespace nvfuser {
 // are equivelent!
 class ReductionParams : public HeuristicParams {
  public:
+  // for combined reduction, final step outer reduction domain is split by this
+  // factor and the reaming is paralled by TIDy
+  int64_t split_factor_final_outer_reduction = 1;
```

Comment:
Move this to the below group of parameters for combined reductions

Reply:
done